cloudformation/ssm-commands/AEM-ManageService.yaml

```diff
@@ -11,6 +11,14 @@ parameters:
       - start
       - stop
 
+  aem:
```

Comment:
Please rename `aem` to `aem_id` for consistency with aem-aws-stack-builder's multiple architecture.
At the moment, `author` and `publish` are good enough which covers both consolidated and full-set architectures.

Reply:
aem_id is not possible, cloudformation only accept alphanumeric characters

Cloudformation Error:

CREATE_FAILED | AWS::SSM::Document | ManageService | Parameter name "aem_id" is not alpha-numeric.
-- | -- | -- | --

Maybe using of aemid could be a alternative?

Reply:
Yup, `aemid` is fine too, it's descriptive enough.